builtin/mainmenu/common.lua

```diff
@@ -63,20 +63,61 @@ function image_column(tooltip, flagname)
 end
 
 --------------------------------------------------------------------------------
+local OWN_CONTINENT = "EU" -- TODO
+local latency_matrix = {
+	["AF"] = { ["AS"]=420, ["EU"]=160, ["NA"]=280, ["OC"]=450, ["SA"]=360, },
+	["AS"] = { ["EU"]=190, ["NA"]=170, ["OC"]=120, ["SA"]=300, },
+	["EU"] = { ["NA"]=120, ["OC"]=380, ["SA"]=200, },
+	["NA"] = { ["OC"]=200, ["SA"]=130, },
+	["OC"] = { ["SA"]=330, },
+}
+
+function estimate_favorite_latency(fav)
+	if OWN_CONTINENT == nil then return nil end
```

Comment:
```suggestion
	if not OWN_CONTINENT then
		return nil
	end
```

Also are the `== nil` checks necessary? Could `not <var>` be used instead?

Reply:
I prefer being explicit about checking for `nil`
also that code is subject to change anyway

Reply:
Ok, that's fine with me then.